examples/telnetd/telnetd.c

```diff
@@ -127,7 +127,7 @@ static void telnetd_quit(int argc, char **argv)
 
 static void telnetd_parse(FAR char *line, int len)
 {
-  struct ptentry_s *entry;
+  FAR const struct ptentry_s *entry;
```

Comment:
Maybe add `FAR` in `telnetd_session` in the next PR

Reply:
Done.